### DISCUSSION TITLE
Manual installation of ipfs is required

### DISCUSSION THREAD

**Discussion by likhita-8091** [Lilypad-Tech/lilypad#344] [General]
![image](https://github.com/Lilypad-Tech/lilypad/assets/44855157/b3220bca-8cb6-4caf-8432-8c5d01f7a2ec)

**Comment by bgins** [comment]
Hey @CodingJzy! 👋 

Apologies for the delay responding. This issue might be from Bacalhau removing support for embedded IPFS: https://github.com/bacalhau-project/bacalhau/issues/3816. Do you recall which version of Bacalhau you were using?

The version that removed embedded IPFS (`v1.4.0`) released one week ago, so this might not be the issue. Could you try Bacalhau `v1.3.2` and see if that works?

We are asking folks to pin to Bacalhau `v1.3.2` while we work on an alternative approach to storage and delivering results. See https://github.com/Lilypad-Tech/lilypad/issues/157 if you would like to follow along. 🙂
